engine/src/main/resources/templates/ImageBuilder/launchSuccessInstantEmailTitleV2.txt

```diff
@@ -0,0 +1 @@
+Instant notification - successful image launch - Red Hat Enterprise Linux
```

Comment:
Should be "Instant notification - Successful image launch - Image builder - Red Hat Enterprise Linux"